view/main/index.css

```diff
@@ -80,21 +80,17 @@ body {
   color: inherit;
   text-decoration: none;
 
-  &:hover,
-  &.is-hover {
-    color: var(--accent);
-    text-decoration: underline;
-  }
-
   &:focus {
     text-decoration: none;
     border-radius: 2px;
     outline-offset: 2px;
   }
 }
 
-.main_logo-path {
-  fill: var(--surface-ui-regular);
+.main:has(.main_logo:hover) .is-em,
```

Comment:
Can you specify element of `.is-em`?

In our version of BEM any modifier must be used together with element class (for the case if later we will add some different element with the same modifier).

Reply:
Yeah, okay. Thought about it. But I've already made my own order in other commits, so I was afraid that such little things would be completely tactless, so I just left original class